modules/common/src/main/scala/notebook/front/widgets/package.scala

```diff
@@ -506,6 +506,16 @@ package object widgets {
                                                   "width" → sizes._1, "height" → sizes._2)))
   }
 
+  case class CustomC3Chart[C:ToPoints:Sampler](originalData:C, chartOptions :String = "{}", override val sizes:(Int, Int)=(600, 400), maxPoints:Int = 25) extends Chart[C] {
```

Comment:
cool! however wouldn't be better to allow passing a `JsValue` that defaults to `Json.obj()`?